book/content/reproducible_environments/02/package-management.md

```diff
@@ -155,7 +155,8 @@ This is the best way to install packages from within Conda as it will also insta
 pip install scipy
 ```
 
-still works.
+still works as long as `pip` is installed inside the currently active conda environment.
```

Comment:
```suggestion
will list 'scipy' package explicitely - as long as `pip` is installed inside the currently active conda environment.
```

Reply:
I am still not sure in pip was mentioned for installing a package in the conda environment, or independently. Usually, it is not recommended to use pip inside conda as it leads to an environment that is not easily reproducible. May be it is worth mentioning that here (ref: https://www.anaconda.com/using-pip-in-a-conda-environment/).

Reply:
But oftentimes the conda version of a package is outdated since the updates are released on pip and someone has to put extra effort to create the conda package. I agree it is worth mentioning the possible incompatibility issues that can arise by mixing pip and conda packages.